plugins/nanospell/plugin.js

```diff
@@ -543,14 +558,39 @@
 					if (!uniqueWords[word] && self.validWordToken(word) && (typeof(spellcache[word]) === 'undefined')) {
 						words.push(word);
 						uniqueWords[word] = true;
-						if (words.length >= max) {
-							return words;
-						}
+						//if (words.length >= max) {
+						//	return words;
+						//}
```

Comment:
Hmm. Probably not. It's a bit of a reminder that I removed the max words per request which wasn't really working, but I suspect I won't need to bring it back.